_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
-The sharing of the RelationshipAttribute by the peer with another peer is technically documented by the creation of a so-called third party owned RelationshipAttribute.
+<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/68fbda7f-30c1-40d1-853a-fd94a91988e8" id="oSgEzd9YW_TQ"></iframe></div>
+
+In the wallet of the peer with whom the underlying RelationshipAttribute of the own shared RelationshipAttribute was shared, a corresponding peer shared RelationshipAttribute is created.
+It is also referred to as a peer shared ThirdPartyRelationshipAttribute.
+It can therefore be interpreted as the counterpart of an own shared ThirdPartyRelationshipAttribute.
+The `shareInfo.sourceAttribute` property of a peer shared ThirdPartyRelationshipAttribute is always undefined, as the own shared RelationshipAttribute used as the source can only be available locally to the `owner` of the RelationshipAttribute.
 
 #### Third party owned RelationshipAttributes
 
-[RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists.
-If the `owner` of the RelationshipAttribute is not the one who shares it, but the peer who has established the Relationship with the `owner` in which the RelationshipAttribute exists, this corresponds to sharing an underlying RelationshipAttribute of a peer shared RelationshipAttribute.
-Such sharing leads to the creation of two certain [LocalAttributes]({% link _docs_integrate/data-model-overview.md %}#localattribute), the so-called third party owned RelationshipAttributes.
+Although the peer is not its `owner`, the underlying [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) of a [peer shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists if its `confidentiality` is not `"private"`. When the peer who has established the Relationship with the `owner` in which the RelationshipAttribute exists shares it, this leads to the creation of two certain [LocalAttributes]({% link _docs_integrate/data-model-overview.md %}#localattribute), the so-called third party owned RelationshipAttributes.
+
+<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/75177324-b871-4516-b68c-84b106682f81" id="vreEz94.RB0c"></iframe></div>
```

Comment:
I'd probably swap the pictures of owner and peer, i.e. the owner is an organization and the peer an App user. On the one hand, I think it is more common that App users have Relationships to organizations instead of organizations having Relationships to other organizations (although that is possible of course). On the other hand, that wouldn't suggest that organizations share your data with third parties using enmeshed.